packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

```diff
@@ -0,0 +1,383 @@
+
+    `*` Required properties
+
+#### Integration variables
+
+Continue with the OIN Wizard and configure integration variables if your URLs are dynamic for each tenant. The variables are for your customer admins to add their specific tenant values during installation.
+
+<StackSnippet snippet="variable-desc" />
+<br>
+
+1. Click the **Integration variables** dropdown arrow and specify the name and label for each variable:
+
+    | <div style="width:100px">Property</div> | Description  |
+    | --------------- | ------------ |
+    | **Label** `*`  | A descriptive name for the dynamic variable that admins see when they install your app integration. |
+     | **Name** `*`  | Specify the variable name. This variable name is used to construct the dynamic URL. It's hidden from admins and is only passed to your external application.<br>String is the only variable type supported.<br>The maximum field length is 1024 characters. |
+
+     `*` This section is optional, but if you specify a variable, both `Label` and `Name` properties are required.
+
+1. Click the **+** icon to add another variable. You can add up to three variables.
+
+1. If you need to delete a variable, click the ![trash can; delete icon](/img/icons/delete_can.png) icon next to it.
+
+ > **Note**: A variable can include a complete URL (for example, `https://example.com`). This enables you to use more globally useful variables such as `${org.baseURL}`.
+
+#### <StackSnippet snippet="protocol-name" inline/> properties
+
+Continue with the OIN Wizard and configure your protocol settings:
+
+1. Click the **<StackSnippet snippet="protocol-name" inline/> properties** dropdown arrow and specify the following protocol properties:
+
+    <StackSnippet snippet="protocol-properties" />
+
+1. Click **Get started with testing** to save your edits and move to the testing experience where you need to enter your integration test details.
+
+### Enter test information
+
+From the OIN Wizard **Test your integration experience** section, specify information required for testing your integration. An OIN analyst uses this information to verify your integration after submission.
+
+#### Test information
+
+A dedicated test admin account in your app is required for Okta integration testing. This test account needs to be active beyond the submission period in case Okta needs to update or troubleshoot your app integration. Ensure that the test admin account has:
+
+* Privileges to SSO configuration in your test app
+* Privileges to administer test users in your test app
+
+See [Test account guidelines](/docs/guides/submit-app-prereq/main/#test-account-guidelines).
+
+> **Note**: This admin account is in your app and not an account in Okta.
+
+1. Click the **Testing information** dropdown arrow and specify the following test account details:
```

Comment:
Should be automatically expanded on load in our current implementation, with no need to click the dropdown arrow